Dockerfile

```diff
@@ -1,31 +1,26 @@
-FROM golang:1.8-alpine
+FROM golang:1
 
 ENV PROJECT=methode-article-image-set-mapper
-COPY . /${PROJECT}-sources/
+ENV BUILDINFO_PACKAGE="github.com/Financial-Times/service-status-go/buildinfo."
 
-RUN apk --no-cache --virtual .build-dependencies add git \
-  && ORG_PATH="github.com/Financial-Times" \
-  && REPO_PATH="${ORG_PATH}/${PROJECT}" \
-  && mkdir -p $GOPATH/src/${ORG_PATH} \
-  # Linking the project sources in the GOPATH folder
-  && ln -s /${PROJECT}-sources $GOPATH/src/${REPO_PATH} \
-  && cd $GOPATH/src/${REPO_PATH} \
-  && BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo." \
-  && VERSION="version=$(git describe --tag --always 2> /dev/null)" \
-  && DATETIME="dateTime=$(date -u +%Y%m%d%H%M%S)" \
-  && REPOSITORY="repository=$(git config --get remote.origin.url)" \
-  && REVISION="revision=$(git rev-parse HEAD)" \
-  && BUILDER="builder=$(go version)" \
-  && LDFLAGS="-X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
-  && echo "Build flags: $LDFLAGS" \
-  && echo "Fetching dependencies..." \
-  && go get -u github.com/kardianos/govendor \
-  && $GOPATH/bin/govendor sync \
-  && go build -ldflags="${LDFLAGS}" \
-  && mv ${PROJECT} /${PROJECT} \
-  && apk del .build-dependencies \
-  && rm -rf $GOPATH /var/cache/apk/*
+COPY . /${PROJECT}/
+WORKDIR /${PROJECT}
 
+# Build app
+RUN VERSION="version=$(git describe --tag --always 2> /dev/null)" \
+ && DATETIME="dateTime=$(date -u +%Y%m%d%H%M%S)" \
+ && REPOSITORY="repository=$(git config --get remote.origin.url)" \
+ && REVISION="revision=$(git rev-parse HEAD)" \
+ && BUILDER="builder=$(go version)" \
+ && LDFLAGS="-s -w -X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+ && echo "Build flags: ${LDFLAGS}" \
+ && CGO_ENABLED=0 go build -a -o /artifacts/${PROJECT} -ldflags="${LDFLAGS}"
```

Comment:
Pass `-mod=readonly` to `go build` in order to prevent accidental update of `go.mod` or `go.sum`.